src/main/java/com/powsybl/openloadflow/dc/WoodburyEngineRhsModifications.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2024, Coreso SA (https://www.coreso.eu/) and TSCNET Services GmbH (https://www.tscnet.eu/)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.openloadflow.dc;
+
+import com.powsybl.math.matrix.DenseMatrix;
+import com.powsybl.openloadflow.network.impl.PropagatedContingency;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**$
+ * @author Pierre Arvy {@literal <pierre.arvy at artelys.com>}
+ */
+public class WoodburyEngineRhsModifications {
```

Comment:
Javadoc missing here to explain why RHS could be overridden. And it is working without the DC sensitivity analysis methods that fill this class?

Reply:
Indeed, thank you. Documentation has been added.